CONTRIBUTING.rst

```diff
@@ -161,6 +161,7 @@ their individual contributions.
 * `Buck Evan, copyright Google LLC <https://github.com/bukzor>`_
 * `Charles O'Farrell <https://www.github.com/charleso>`_
 * `Charlie Tanksley <https://www.github.com/charlietanksley>`_
+* `Chase Garner <https://www.github.com/chasegarner>` _ (chase@garner.red)
```

Comment:
nit: lose the space before the underscore, or this won't render as a link correctly.

(I don't know if the rst-lint job will spot this – what you’ve written is valid RST, it's just not what you want!)